source/agora/consensus/ValidatorSet.d

```diff
@@ -103,16 +104,16 @@ public class ValidatorSet
 
         try
         {
-            static ubyte[] buffer;
-            serializeToBuffer(enroll, buffer);
-            const img = PreImageInfo(enroll.utxo_key, enroll.random_seed, 0);
-            static ubyte[] pbuffer;
-            serializeToBuffer(img, pbuffer);
-
             () @trusted {
+                // todo: we probably don't need to store enrolled_height here
+                // as it's only used for *this* validator
```

Comment:
Can you expand on this ?  Enrolled height is used to tell when we need to remove a validator, isn't it ?

Reply:
Ah right, brain slip. I'll remove this.